Cargo.toml

```diff
@@ -20,6 +20,9 @@ name = "rls"
 test = false
 path = "rls/src/main.rs"
 
+[patch.crates-io]
+rustfmt-nightly = { git = "https://github.com/rust-lang/rustfmt", rev = "06fc39f905261d2e5e8f60b4b8cc1e11db9cf177" }
```

Comment:
Has to be patched until Rustfmt is released containing https://github.com/rust-lang/rustfmt/pull/3424.